packages/govuk-frontend/src/govuk/components/character-count/character-count.mjs

```diff
@@ -70,7 +71,11 @@ export class CharacterCount extends GOVUKFrontendComponent {
     super()
 
     if (!($module instanceof HTMLElement)) {
-      return this
+      throw !$module
+        ? new MissingElementError('Character Count: $module not found')
+        : new TypeError(
+            'Character Count: $module is not an instance of "HTMLElement"'
+          )
```

Comment:
How about a non-blocking base class idea to solve these suffixes?

E.g. New getter `.log` to automatically suffix `Character count: `

```mjs
this.log = new Logger('Character count')

this.log.debug()
this.log.warn()
this.log.error()
```

Except it can also put its log messages into `this.log.throw()` for neater things like:

```suggestion
      this.logger.throw(
        $module ? TypeError : MissingElementError,
        $module
          ? '$module is not an instance of "HTMLElement"'
          : '$module not found'
      )
```

Reply:
Can we track that option as part of #4072 as this relates more to this aspect of the errors. One thing that comes to mind here is whether any logging through that class would be able to be dropped by minifiers similarly to how they can drop `console.<logLevel>`. It does open a line of thought where the prefixing does not happen in the error classes, though, so that's definitely worth tracking 😊

Reply:
Yeah not a blocker

We'd have the same [`drop_console`](https://terser.org/docs/options/#compress-options) problem anywhere but the error "throw" would still be kept

Just an idea to save a few bytes with `new Logger('Character count')` keeping the string safe